tern/formats/json/html.py

```diff
@@ -0,0 +1,375 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017-2020 VMware, Inc. All Rights Reserved.
```

Comment:
Small nit but this should just be `Copyright (c) 2020` since that is the year the file originated.

Reply:
ok sure :)